lua/ttt2/libraries/database.lua

```diff
@@ -1092,14 +1092,18 @@ if SERVER then
 				return true, value
 			end
 
+			ConvertTable(sqlData, accessName)
+
 			local newTable = {}
 			for _key in pairs(dataTable.keys) do
+				-- Get default values if no value was saved
+				if sqlData[_key] == nil then
```

Comment:
is there a reason for this explicit check? instead of `if not xx then`? Can this value be `false`

Reply:
Yes it can be false. This can be any datatype.